editors/vscode/slint.tmLanguage.json

```diff
@@ -190,6 +186,18 @@
                 {
                     "match": "\\b\\d+(\\.\\d*)?(%|px|phx|pt|in|mm|cm|ms|s|deg|rad|turn)?\\b",
```

Comment:
this is still missing rem.

I think your screenshot is biased because it also run the semantic token from the LSP